packages/api2/src/database/migrations/13_settings_require_discord_role.ts

```diff
@@ -1,5 +1,5 @@
-import { SettingGroup } from '@/settings/types';
-import { SettingsModel } from '../../settings/entities';
+import { SettingGroup } from '@/settings/interfaces/settings-group.interface';
```

Comment:
Absolute imports in migrations using `@` did not work for me when running `yarn workspace api2 start`. Did it work for you?